livekit-agents/livekit/agents/pipeline/pipeline_agent.py

```diff
@@ -1206,7 +1214,6 @@ def on_human_start_of_speech(self, ev: vad.VADEvent) -> None:
         self._last_recv_start_of_speech_time = time.perf_counter()
         if self.validating:
             assert self._validating_task is not None
-            self._validating_task.cancel()
```

Comment:
I don't think we should cancel it?

Reply:
yeah this should be kept.. reverting this deletion